### ISSUE TITLE
Dara theme: Block post block showing Project. Issue with "read more" component.

### ISSUE THREAD

**Issue by ezebecke** [Automattic/themes#7697]
### Quick summary

When the block post block is set to display projects and the button "Load more posts" is enabled. All the subsequent loaded projects don't have the "read more" link

### Steps to reproduce

Use "Dara" theme on a Simple site. 
Add blog post block on a new page.
Set the blog to display only projects and enable "Read more" and "Load more posts" button.
Check loading more posts on the published page.

### What you expected to happen

To see "keep reading" links on all the loaded posts (projects)

### What actually happened

The projects that are initially loaded have the read more links:

![CleanShot 2024-03-21 at 16 53 42](https://github.com/Automattic/themes/assets/24919469/f9749aca-c55c-4238-8ff0-2731bb0407fb)

After pressing the button, subsequent projects don't have the read more links

![CleanShot 2024-03-21 at 16 54 39](https://github.com/Automattic/themes/assets/24919469/9bb82ccd-ec87-4c11-ae0a-b2aaca79bf9f)
 read more link: 

### Browser

Google Chrome/Chromium

### Context

7890694-zd

### Platform (Simple, Atomic, or both?)

Simple

### Other notes

Only happening on Simple Sites.

### Reproducibility

Once

### Severity

One

### Available workarounds?

No but the platform is still usable

**Comment by maciejpilarski** [comment]
📌 REPRODUCTION RESULTS
– Tested on Simple – Replicated 

📌 FINDINGS/SCREENSHOTS/VIDEO
I was able to replicate the issue.

📌 ACTIONS
– Triaged

**Comment by mmtr** [comment]
I don't think this is an issue of the Dara theme, since I can replicate with other themes like Batch or Twenty Twenty-Five.
